src/convertToObject.js

```diff
@@ -1,12 +1,68 @@
 'use strict';
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString === '') {
+    return {};
+  }
+
+  const firstObrabotka = sourceString;
+
+  const bigArr = firstObrabotka.split(';');
```

Comment:
The variable 'bigArr' is ambiguously named. Use more descriptive names that convey the contents or purpose of the array, such as 'styleDeclarations'.